DEV_ENV.md

```diff
@@ -3,10 +3,13 @@
 ## Development quickstart
 
 1. [install docker](https://docs.docker.com/get-docker/). If brew is installed run `brew install docker`. If you have a Mac, [install Docker Desktop](https://www.docker.com/products/docker-desktop) and open it so it's running on your machine. Note: If you have a Mac M1 (arm64 CPU), follow instructions in [`docker-compose.yml`](docker-compose.yml) under `oidc` service, for manually building the oidc-server-mock images for the M1 architecture.
-1. If you have a Mac with an M1 chip, please add `export DOCKER_DEFAULT_PLATFORM="linux/amd64"` to your `.zshrc` or `.bashrc` file (don't forget to `source ~/.zshrc` or restart the terminal). Otherwise, Docker will fail to build your images.
+1. [install jq](https://stedolan.github.io/jq/download/). If brew is installed run `brew install jq`.
+1. [install postgres](https://formulae.brew.sh/formula/postgresql@14). If brew is installed run ` brew install postgresql@14` (or whatever is the latest stable release at the time you are reading this). For the purposes of local development, the main reason install `postgres` is to have
```

Comment:
You want `brew install libpq` here.

Reply:
@ebezzi - I will add this in a following PR. This got merged automatically (that is I didn't do it) which is weird